src/test/exploit_tests.cpp

```diff
@@ -900,12 +900,9 @@ BOOST_AUTO_TEST_CASE(thinblock_tests)
         std::unique_ptr<CNode> dummyNode9Mem(new CNode(INVALID_SOCKET, addr4, "", true));
         CNode &dummyNode9 = *dummyNode9Mem;
 
-        // The number of tx bytes in this block is 3784 bytes.  In order for the node to be disconnected
-        // we have to make the maxAllowedSize be less than 3784/maxMessageSizeMultiplier (maxMessageSizeMultiplier =
-        // 16).
-        // Therefore the excesssiveBlockSize must be less than 236.5 inorder to trigger an oversized block and susequent
-        // disconnection.
-        excessiveBlockSize = 234;
+        // The excessive block size is used as the limit for the maximum sum of bytes for
+        // all currently processing thinblocks.
+        excessiveBlockSize = 100;
```

Comment:
same here, plus before we moved from 234 to 80 and here from 234 to 100?

Reply:
same